app/helpers/rubygems_helper.rb

```diff
@@ -16,11 +16,11 @@ def link_to_page(id, url)
   end
 
   def link_to_github(rubygem)
-    if !rubygem.linkset.code.nil? && URI(rubygem.linkset.code).host == "github.com"
-      URI(@rubygem.linkset.code)
-    elsif !rubygem.linkset.home.nil? && URI(rubygem.linkset.home).host == "github.com"
-      URI(rubygem.linkset.home)
+    result = [rubygem.linkset.code, rubygem.linkset.home].detect do |linkset|
```

Comment:
Looks much better :+1:
Can you please update it to use `Links`? You will need to pass `@latest_versions` as well from the view and use
`rubygem.links(version)` here to get links.

Reply:
Will do -- I'm holding on this only because it isn't terribly important to merge :)